ui/src/app/shared/components/orb/sink/sink-details/sink-details.component.scss

```diff
@@ -26,6 +26,15 @@ input {
   &idle {
     color: #f2994a;
   }
+  &provisioning {
+    color: #3089fc;
+  }
+  &provioning_error {
+    color: #df316f;
+  }
+  &warning {
+    color: #f2c94c;
+  }
```

Comment:
should these be defined here since they're in _overrides.scss too?

Reply:
makes sense!